src/Polysemy/Internal/Scoped.hs

```diff
@@ -7,10 +7,9 @@ import Data.Kind (Type)
 
 import Polysemy
 
--- | @Scoped@ transforms a program so that @effect@ is associated with a
--- @resource@ within that program. This requires the interpreter for @effect@ to
--- be parameterized by @resource@ and constructed for every program using
--- @Scoped@ separately.
+-- | @Scoped@ transforms a program so that an interpreter for @effect@ may
+-- perform arbitrary actions, like resource management, before and after the
+-- computation wrapped by a call to 'scoped' is executed.
```

Comment:
Now that `resource` doesn't exist for disambiguation between `Scoped`s, perhaps we should remind people of `Tagged`(?)

Reply:
not sure what scenario you're thinking of here. can you formulate something?

Reply:
Actually, considering the resource typed param should be kept polymorphic people couldn't disambiguate using it anyway. So nevermind.